pkg/provider/provider.go

```diff
@@ -611,10 +611,15 @@ func (p Provider) buildUDPEntrypoint(svc *topology.Service, port int32) (string,
 func (p *Provider) buildHTTPServiceFromService(t *topology.Topology, svc *topology.Service, scheme string, port int32) *dynamic.Service {
 	var servers []dynamic.Server
 
+	containMissingPod := false
+
 	for _, podKey := range svc.Pods {
 		pod, ok := t.Pods[podKey]
 		if !ok {
-			p.logger.Errorf("Unable to find Pod %q", podKey)
```

Comment:
In my opinion, it's better to add more information to the current log statement than to add an extra logic. Something like the following:

```
p.logger.Errorf("Unable to find Pod %q for HTTP service %s@%s", podKey, topology.Key{Name: svc.Name, Namespace: svc.Namespace})
```

Reply:
Hi, thanks for your review.

Updated this PR to reduce logging logic.